python/cudf/cudf/tests/test_series.py

```diff
@@ -655,4 +655,4 @@ def test_series_error_equality(sr1, sr2, op):
     gsr1 = cudf.from_pandas(sr1)
     gsr2 = cudf.from_pandas(sr2)
 
-    assert_exceptions_equal(op, op, [sr1, sr2], [gsr1, gsr2])
+    assert_exceptions_equal(op, op, ([sr1, sr2], {}), ([gsr1, gsr2], {}))
```

Comment:
This signature is really powerful but a bit cumbersome now. Any ideas on how we could make it easier to use without making it confusing?

Some thoughts I have are what if we handled length 1 tuples where if the only element is a sequence then we treat it as positional args, if the only element is a mapping then we treat it as kwargs. That would make this call:
```
assert_exceptions_equal(op, op, ([sr1, sr2],), ([gsr1, gsr2],))
```

Which is a little bit less cumbersome.

Reply:
Regardless we can handle this in a follow up though

Reply:
Fixed this as part of simplifying variable names